README.md

```diff
@@ -23,14 +23,79 @@
 
 This is a VS Code extension which enables the interactive debugging of DFDL schema parsing using [Apache Daffodil](https://daffodil.apache.org/).
 
+## Build Requirements
+- SDK 8 or higher
+- SBT 0.13.8 or higher
+- Node 10 or higher
 
-## Installation
+## Installation - via release
 
 Until the extension is available in the [VS Code Extension Marketplace](https://marketplace.visualstudio.com/vscode), please download the latest `.vsix` file from the [releases page](https://github.com/apache/daffodil-vscode/releases). Then install it by either:
   * using the "Extensions: Install from VSIX" command from within VS Code (open the Command Palette with Shift-Command-P, then type `vsix` to bring up the command and pointing it at the downloaded `vsix` file); or
   * on the command-line via `code --install-extension <path-to-downloaded-vsix>`.
 
 
+## Installation - via source
+
+:exclamation: Currently only Daffodil 3.1.0 is support, will be updated in the future :exclamation:
+
+Firstly clone the repo:
+* apache/daffodil-vscode (https://github.com/apache/daffodil-vscode)
+
+Backend - Debugger:
+* Note down the current version listed in `package.json`, eg `1.0.0`
+  * Referenced later as `${package.json_version}`
+* Note what the dapodil data folder location for your OS is.
+  * Linux = `/home/USERNAME/.local/share/dapodil`
+  * Mac = `/Users/USERNAME/Library/Application\ Support/dapodil`
+  * Windows = `C:\\Users\\USERNAME\\AppData\\Roaming\\dapodil`
+
+  :exclamation: Ensure this folder location exists :exclamation:
+
+* Create daffoil debugger zip
+  ```bash
+  sbt universal:packageBin
+  ```
+  * Package will be at
+    ```
+    server/core/target/universal/daffodil-debugger-3.1.0-${package.json_version}.zip
+    ```
+    eg
+    ```
+    server/core/target/universal/daffodil-debugger-3.1.0-1.0.0.zip
+    ```
+
+* Unzip debugger zip in data folder
+  ```
+  unzip server/core/target/universal/daffodil-debugger-3.1.0-${package.json_version}.zip -d ~/Library/Application\ Support/dapodil
+  ```
+
+Extension:
+
+:exclamation: This requires there to be a release with the version specified in `package.json` or for the steps for `Backend - Debugger` have been completed. :exclamation:
+
+* Install
+  ```bash
+  yarn install
+  ```
+
+* Compile
+  ```bash
+  yarn compile
+  ```
+
+* Package
+  ```bash
+  yarn package
+  ```
+
+  * Creates file `daffodil-debugger-${package.json_version}.vsix`
```

Comment:
This is minor, but the heavy use of lists and nesting I find a bit hard to follow. I would suggest using sections and subsections rather than lists and sublists.